<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>751</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>测量模式切换</string>
    </property>
    <widget class="QPushButton" name="btnVnaModeSwitch">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>92</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>VNA</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>751</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>连接设备</string>
    </property>
    <widget class="QLineEdit" name="leDeviceIp">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>271</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>169.254.227.100</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConnect">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>92</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>连接</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>设备ip</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>761</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>返回信息</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>741</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnCore">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>370</y>
      <width>92</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>记录</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>751</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>频率设置</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>起始频率</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>终止频率</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbStartFreqUnit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>89</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Hz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>kHz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MHz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GHz</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btnSetStartFreq">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>92</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>设置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnQueryStopFreq">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>60</y>
       <width>92</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnQueryStartFreq">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>92</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSetStopFreq">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>92</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>设置</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsbStartFreq">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsbStopFreq">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbStopFreqUnit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>89</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Hz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>kHz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MHz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GHz</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>751</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>迹线读取</string>
    </property>
    <widget class="QPushButton" name="btnReadFormatData">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>92</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>读取格式化数据</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>92</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>test</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
